### ISSUE TITLE
runtime error: slice bounds out of range Polkadot Block 4383090

### ISSUE THREAD

**Issue by infinitedaremo** [itering/scale.go#24]
We ran into this issue when trying to parse the above block. Using the latest commit 

```
runtime error: slice bounds out of range [:-5919563810157205112]
/usr/local/go/src/runtime/panic.go:106 (0x40378e4)
        goPanicSliceAcap: panic(boundsError{x: int64(x), signed: true, y: y, code: boundsSliceAcap})
/Users/gavinwoods/go/pkg/mod/github.com/itering/scale.go@v1.0.13/types/scaleBytes.go:18 (0x483a059)
        (*ScaleBytes).GetNextBytes: data := s.Data[s.Offset : s.Offset+length]
/Users/gavinwoods/go/pkg/mod/github.com/itering/scale.go@v1.0.13/types/base.go:69 (0x4839ec1)
```

**Comment by yrong** [comment]
you may check if [types](https://github.com/itering/subscan-essentials/blob/master/configs/source/polkadot.json) match first

**Comment by infinitedaremo** [comment]
Im having this issue because I followed the solution in this issue thread: https://github.com/itering/scale.go/issues/21

Whilst that does fix the issue with the block 4230783 it causes issues with block 4383090.

Not having the keys type does however fix the latest block.

How do i proceed?

**Comment by freehere107** [comment]
@infinitedaremo   polkadot.json add ```"ValidatorPrefs#28-?": "ValidatorPrefsWithBlocked"```
